Sources/Amplitude/SSLCertificatePinning/ISPCertificatePinning.m

```diff
@@ -120,9 +120,11 @@ + (BOOL)verifyPinnedCertificateForTrust:(SecTrustRef)trust andDomain:(NSString*)
 
         NSArray *anchorArray = [NSArray arrayWithObject:(__bridge id)(anchorCertificate)];
         if (SecTrustSetAnchorCertificates(trust, (__bridge CFArrayRef)(anchorArray)) != 0) {
+            SecTrustSetAnchorCertificates(trust, NULL);
             CFRelease(anchorCertificate);
             break;
         }
+        SecTrustSetAnchorCertificates(trust, NULL);
```

Comment:
maybe you just need to do this one time above line 122?

Reply:
No, it has to happen after `SecTrustSetAnchorCertificates(trust, (__bridge CFArrayRef)(anchorArray))` in the if condition.